sasl-doc/sasl.scrbl

```diff
@@ -106,6 +138,24 @@ protocol is done}
 ]
 }
 
+@defthing[
+  #:kind "contract"
+  sasl-next/c contract?
+  #:value (or/c 'done
+                (-> any/c
+                    (or/c bytes? string?)
+                    (values (or/c #f bytes? string?) sasl-next/c)))
+]{
+  The contract for custom SASL mechanism state transition procedures.
+}
+
+@defstruct[(exn:fail:sasl:fatal? exn:fail?)
```

Comment:
These should be the struct names, not the predicates.

Reply:
Whoops! Will fix in a sec, thanks.